project/Dependencies.scala

```diff
@@ -21,7 +21,7 @@ object Dependencies {
   val serviceTestV = s"5.0-$workbenchLibsHash"
   val workbenchModelV = s"0.20-$workbenchLibsHash"
   val workbenchGoogleV = s"0.32-$workbenchLibsHash"
-  val workbenchGoogle2V = s"0.36-$workbenchLibsHash"
+  val workbenchGoogle2V = s"0.4-39b4ef35-SNAP"
```

Comment:
Will use the official one once this is merged https://github.com/broadinstitute/workbench-libs/pull/1726, I just published from my workbenchlibs branch for testing purposes